airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -1209,7 +1209,9 @@ def add_to_outputs(
                     quoted_unique_key=self.get_unique_key(in_jinja=True),
                     active_row_column_name=active_row_column_name,
                     normalized_at_incremental_clause=self.get_incremental_clause_for_column(
-                        "this.schema + '.' + " + self.name_transformer.apply_quote(final_table_name),
+                        self.name_transformer.apply_quote("this.schema", literal=False)
```

Comment:
nit: please use f-strings instead of concatenation. It makes code more readable.

Reply:
I have improved readability with string formatting